Utils/Dataflow/009_oracleConnector/Oracle2JSON.py

```diff
@@ -53,6 +56,8 @@ def main():
 
     # Offset timezone initialization
     init_offset_tz(config)
+    # Offset delay initialization
+    init_offset_delay(config)
```

Comment:
May it be `init_offset_params(config)` instead of these two calls?

Reply:
I thought about this before making the changes, but decided against it. Maybe in the future we will actually implement the suggestion described above, about moving the whole config into global variable, but I think we should start doing it when we make the decision, not now.